admin/pages/whats-new.php

```diff
@@ -1,222 +1,87 @@
-                             title="Step 2: Tailor the look & feel to suit your website"/>
-
                         <div class="text">
+                            <?php 
+                                $step2 = plugins_url(). '/'. WMP_DOMAIN . "/admin/images/step2.png";
+                            ?>
+                            <img src="<?php echo $step2; ?>" title="Step 2: Tailor the look & feel to suit your website" />
                             <span class="title">Step 2: Tailor the look & feel to suit your website</span>
 
-                            <p>Tailor the look & feel by choosing from the available color schemes, various images, adding your logo and app icon.</p>
+                            <p>
+                                Tailor the look & feel by choosing from the available color schemes, various images, adding your logo and app icon.
+                            </p>
                         </div>
                     </div>
                     <div class="spacer-0"></div>
 
                     <div class="feature left">
-                        <?php
-
-                        $step3 = get_home_url()."\wp-content\plugins\wordpress-pwa\admin\images\step3.png";
-                        
-                        ?>
-                        <img src="<?php echo $step3; ?>"
-                             title="Step 3. Decide on the content to be included in your layout"/>
 
                         <div class="text">
+                            <?php
+                                $step3 = plugins_url(). '/'. WMP_DOMAIN . "/admin/images/step3.png";
+                            ?>
+                            <img src="<?php echo $step3; ?>" title="Step 3: Decide on the content to be included in your layout" />
                             <span class="title">Step 3: Decide on the content to be included in your layout</span>
 
-                            <p>Initially all content will be activated to show on your mobile web application. Decide between various pages & categories to be activated/deactivated, by simply clicking on the row & selecting to activate/deactivate it. Categories & pages can also be rearranged by dragging the corresponding row to the desired position.</p>
+                            <p>
+                                Initially all content will be activated to show on your mobile web application. Decide between various pages & categories to be activated/deactivated, by simply clicking on the row & selecting to activate/deactivate it. Categories & pages can also be rearranged by dragging the corresponding row to the desired position.
+                            </p>
                         </div>
                     </div>
                     <div class="spacer-0"></div>
-                    <?php 
-
-                     $step4 = get_home_url()."\wp-content\plugins\wordpress-pwa\admin\images\step4.png";   
 
+                    <?php 

```

Comment:
Not protesting or anything. I just wanted to know how this is more efficient than my way.